More v5 docs! (#19)
RLBot/wiki: docs/v5/botmaking/scripts.md

# Scripts

> Sometimes it can be useful to have access to game values, debug rendering or state setting while not being a bot playing within the match. RLBot has an easy solution for that - scripts!

Reviewer: ```suggestion
Sometimes it can be useful to have access to game values, debug rendering, or state setting while not being a bot playing within the match. RLBot has an easy solution for that - scripts!
```

> Scripts are very similar to normal bots, but instead of having a `bot.toml` they have a `script.toml` file.

Reviewer: ```suggestion
Just like how bots are defined by a `bot.toml`, scripts are defined by a `script.toml`, and the content is very similar.
```

??? example "Example `script.toml` file"
    ```toml
    [settings]
    agent_id = "rlbot-community/script"
    name = "My script"
    run_command = "python script.py"
    run_command_linux = "./venv/bin/python script.py"

    [details]
    description = "Made possible by RLBot"
    fun_fact = "This is a test script"
    source_link = "https://github.com/RLBot/RLBot"
    developer = "RLBot community"
    language = "C#"
    tags = []

>     ```

Reviewer: Example's run_command: "python script.py"
Example's language: "C#"
🥴
